src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -26,14 +26,18 @@
           Condition="'$(StringResourcesPath)'!='' AND '$(OmitResources)'!='true'"
           Inputs="$(StringResourcesPath)"
           Outputs="$(IntermediateResOutputFileFullPath)">
-    
-    
 
     <GenerateResourcesCode
         ResxFilePath="$(StringResourcesPath)" 
         OutputSourceFilePath="$(IntermediateResOutputFileFullPath)" 
         AssemblyName="$(AssemblyName)" />
 
+    <Copy Condition="'$(BuildingUAPVertical)' == 'true'"
+          SourceFiles="$(StringResourcesPath)"
+          DestinationFiles="$(ResourcesFolderPath)\$(AssemblyName).resw"
```

Comment:
@safern already tried all resx files we have in corefx and worked after renaming it to resw. so far this is good enough till we find a case break this assumption.